pkg/cmd/server/origin/controller.go

```diff
@@ -7,17 +7,62 @@ import (
 
 	"github.com/golang/glog"
 
+	kubernetes "github.com/openshift/origin/pkg/cmd/server/kubernetes/master"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/util/cert"
+	kubecontroller "k8s.io/kubernetes/cmd/kube-controller-manager/app"
 	kapi "k8s.io/kubernetes/pkg/api"
-	kubecontroller "k8s.io/kubernetes/pkg/controller"
+	kcontroller "k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/serviceaccount"
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	"github.com/openshift/origin/pkg/cmd/server/crypto"
 	"github.com/openshift/origin/pkg/cmd/server/origin/controller"
 )
 
+func (c *MasterConfig) NewKubernetesControllerInitalizers(kc *kubernetes.MasterConfig) (map[string]kubecontroller.InitFunc, error) {
+	ret := map[string]kubecontroller.InitFunc{}
```

Comment:
I think this function can do the stitching together of the upstream map with our overrides.  Put yourself in the rebasers shoes.  You want the new controllers, plus our existing overrides without having to do anything specific, but you want one map to look at for rebasing purposes.